llm_assistant/ollama/openai_chat.py

```diff
@@ -0,0 +1,21 @@
+from openai import OpenAI
+import argparse
+
+# We don't need an actual api_key here. See `ollama/README.md`
+client = OpenAI(base_url="http://0.0.0.0:8000", api_key="FAKE")
```

Comment:
If we don't need actual API key, please remove `api_key="FAKE"` from the code.

Reply:
Please read the mentioned README file. OpenAI SDK requires the `api_key` param to be present, but since we're interacting with a proxy, it isn't needed. The "FAKE" is kept to signify that it's not a real key.

Reply:
My bad, it is documented in https://github.com/BerriAI/litellm?tab=readme-ov-file#step-2-make-chatcompletions-request-to-proxy